src/leappto/cli.py

```diff
@@ -614,10 +598,7 @@ def print_migrate_info(text):
     elif parsed.action == 'check-target':
         target = parsed.target
 
-        lmp = LibvirtMachineProvider()
-        machines = lmp.get_machines()
-
-        machine_dst = _find_machine(machines, target)
+        machine_dst = _inspect_machine(target)
```

Comment:
Seeing this I think we need to have the custom user support here - That however would be a different PR